config/initializers/omniauth.rb

```diff
@@ -1,4 +1,10 @@
 Rails.application.config.middleware.use OmniAuth::Builder do
-  provider :google_oauth2, ENV['GOOGLE_CLIENT_ID'], ENV['GOOGLE_CLIENT_SECRET']
+  provider :google_oauth2, ENV['GOOGLE_CLIENT_ID'], ENV['GOOGLE_CLIENT_SECRET'],
+  {
+    scope: 'userinfo.email, userinfo.profile, books',
+    prompt: 'select_account',
+    image_aspect_ratio: 'square',
+    image_size: 50
+  }
 end
 OmniAuth.config.allowed_request_methods = [:get]
```

Comment:
@LawrenceWhalen just noticed this line as well - we're going to need to make post and delete requests also for book management